cfme/infrastructure/provider/__init__.py

```diff
@@ -285,6 +283,15 @@ def hosts(self):
             result.append(Host(name=host["name"], credentials=cred))
         return result
 
+    def get_clusters(self):
+        """returns the list of clusters belonging to the provider"""
+        web_clusters = []
+        sel.force_navigate('infrastructure_clusters')
```

Comment:
Doesn't this return _all_ clusters? ie, it doesn't seem to filter at all to show _just_ the providers clusters? Did I miss something?